### DISCUSSION TITLE
Distance From Screen Detection

### DISCUSSION THREAD

**Discussion by desi-dimitrova** [vladmandic/human#245] [General]
Hi Vlad,

Are there any limitations to the size of the image that is being analyzed? I am asking because I see an issue with the distance value coming from the Face Iris model if the video/image input is with resolution 200x200 px. When I test with your demo and set image width & height to 200 without moving closer to the screen or moving back from it, I get 2 different results for each frame - in one frame I get 28.49 and in the next 21.36 without moving at all.. the distance values just keep alternating. I am pretty sure this is due to the resolution being 200x200 because I can't reproduce that issue with higher resolutions but I couldn't find information on input size limitations. Do you have any recommendations for a workaround if the input size is the issue? The goal is to detect how close a person is to the screen even with 200x200 video resolution.

Let me know if you need any other information. 

Thanks!
Desi

**Comment by vladmandic** [comment]
distance is calculated based on the size of iris. and iris detection works on detected face image further cropped to 64px for each eye. and if its less, it is internally upsampled.  

now work backwards - how big is eye compared to entire image?  
if image size is 200px, iris detection has to work with ~10px as input and that is just not enough for any kind of quality.

**Comment by desi-dimitrova** [comment]
Thank you! This cleared things out for me.
